bugbot/rules/webcompat_sightline.py

```diff
@@ -0,0 +1,76 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from typing import Any, Optional
+
+from bugbot import gcp
+from bugbot.bzcleaner import BzCleaner
+
+
+class WebcompatSightline(BzCleaner):
+    WHITEBOARD_ENTRY = "[webcompat:sightline]"
+
+    def __init__(self):
+        super().__init__()
+        self.sightline_ids = set()
+
+    def description(self) -> str:
+        return "Web Compat site report in the sightline metric set"
+
+    def filter_no_nag_keyword(self) -> bool:
+        return False
+
+    def has_default_products(self) -> bool:
+        return False
+
+    def handle_bug(
+        self, bug: dict[str, Any], data: dict[str, Any]
+    ) -> Optional[dict[str, Any]]:
+        bug_id = str(bug["id"])
+        whiteboard = bug["whiteboard"]
+
+        if bug["id"] in self.sightline_ids:
+            if self.WHITEBOARD_ENTRY not in whiteboard:
+                self.autofix_changes[bug_id] = {
+                    "whiteboard": whiteboard + self.WHITEBOARD_ENTRY
+                }
+                return bug
+        elif self.WHITEBOARD_ENTRY in whiteboard:
+            self.autofix_changes[bug_id] = {
+                "whiteboard": whiteboard.replace(self.WHITEBOARD_ENTRY, "")
+            }
+            return bug
```

Comment:
nit: this could improve the readability a bit.
```suggestion
        else:
            if self.WHITEBOARD_ENTRY in whiteboard:
                self.autofix_changes[bug_id] = {
                    "whiteboard": whiteboard.replace(self.WHITEBOARD_ENTRY, "")
                }
                return bug
```